trailblaze-android/src/main/java/xyz/block/trailblaze/android/uiautomator/AndroidOnDeviceUiAutomatorScreenState.kt

```diff
@@ -133,7 +133,10 @@ class AndroidOnDeviceUiAutomatorScreenState(
         if (setOfMarkEnabled && screenshotBitmap != null) {
           viewHierarchy?.let {
             val markedBitmap = addSetOfMark(screenshotBitmap, viewHierarchy)
-            screenshotBitmap.recycle()
+            if (screenshotBitmap != markedBitmap) {
+              // Recycle the original bitmap to free up memory
+              screenshotBitmap.recycle()
+            }
```

Comment:
This is unrelated to this PR, but was an error I hit a few times on a specific device.  The same exact bitmap came back and caused a runtime error.